lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -54,6 +108,9 @@ def export_kymotrackgroup_to_csv(
 
     position = np.hstack([track.position for track in kymotrack_group])
     seconds = np.hstack([track.seconds for track in kymotrack_group])
+    minimum_length = np.hstack(
+        [track._minimum_observable_duration * np.ones(len(track)) for track in kymotrack_group]
+    )
```

Comment:
not any shorter, but wonder if it's more clear than multiplying a vector of ones

```suggestion
    minimum_length = np.hstack(
        [np.full(len(track), track._minimum_observable_duration) for track in kymotrack_group]
    )
```